x-pack/plugins/grokdebugger/public/sections/grokdebugger/components/custom_patterns_input/custom_patterns_input.js

```diff
@@ -33,11 +33,13 @@ app.directive('customPatternsInput', function () {
         $scope.aceLoaded = (editor) => {
           this.editor = editor;
           editor.getSession().setUseWrapMode(true);
+          editor.renderer.setScrollMargin(7, 7, 0, 0);
```

Comment:
What do the numbers correspond to? Might be helpful to make constants for them.

Reply:
Also, what problem does increasing the scroll margin solve? Might be good to add a short explainer comment.

Reply:
I added a comment - basically without the scroll margin we end up with the insertion point pressed right against the border and it's not very pleasing. A thin margin (similar to the GH editor I'm typing into now, but much smaller) provides a nicer experience.